### PR TITLE
Translate specifics/ets.md page to Portuguese - issue #123

### PR DESCRIPTION
**Pull request by psantos10**
ets.md page translated to Portugues.

@pragmaticivan review please

### CODE REVIEW

File: pt/lessons/specifics/ets.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  - [Pesquisa Avançada](#pesquisa-avancada)
+- [Eliminando Dados](#eliminando-dados)
+  - [Removendo Registos](#removendo-registos)
+  - [Removendo Tabelas](#removendo-tabelas)
+- [Exemplo de uso do ETS](#exemplo-de-uso-do-ets)
+- [ETS baseado em Disco](#ets-baseado-em-disco)
+
+## <a name="visao-geral"></a> Visão Geral
+
+ETS é um mecanismo de armazenamento em memória robusto para objetos do Elixir e do Erlang que já vem incluído. ETS é capaz de armazenar grandes quantidades de dados e oferece um tempo constante para acesso aos dados.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: I think "e oferecer um tempo constante" is better.

File: pt/lessons/specifics/ets.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+```
+
+### Tipos de Tabelas
+
+Existem quatro tipos de tabelas disponíveis no ETS:
+
++ `set` — Este é o tipo de tabela padrão. Um valor para cada chave. Chaves são únicas.
++ `ordered_set` — Igual ao `set` mas ordenado por termo Erlang/Elixir. É importante notar que comparação de chave é diferente dentro do `ordered_set`. Chaves não devem coincidir desde que sejam iguais, tanto 1 como o 1.0 são considerados iguais.
++ `bag` — Muitos objetos por cada chave mas apenas uma instância de cada objeto por cada chave.
++ `duplicate_bag` —Muitos objetis por cada chave; chaves duplicadas são permitidas.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: `Muitos objetos`

File: pt/lessons/specifics/ets.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+```elixir
+iex> :ets.lookup(:user_lookup, "doomspork")
+[{"doomspork", "Sean", ["Elixir", "Ruby", "Java"]}]
+```
+
+### <a name="correspondencias-simples"></a> Correspondências Simples
+
+ETS foi construído para o Erlang, logo, tenha em atenção que correspondência de variáveis pode parecer um _pouco_ desajeitado.
+
+Para especificar uma variáveil no nosso *match*, usamos os *atoms* `:"$1"`, `:"$2"`, `:"$3"`, e assim por diante; o número da variável reflecte a posição do resultado e não a possição do *match*. Para valores que não nos interessam usamos a variável `:"_"`.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: replace `reflecte` for `reflete`

Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: replace `possição` for `posição`

File: pt/lessons/specifics/ets.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+
+Valores podem ser usado na correspondênca, mas apenas variáveis se tornarão como parte do nosso resultado. Vamos juntar tudo isso e ver como funciona:
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: `Valores podem ser usados`

File: pt/lessons/specifics/ets.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+
+```elixir
+iex> :ets.match(:user_lookup, {:"$1", "Sean", :"_"})
+[["doomspork"]]
+```
+
+Vamos olhar noutro exemplo para ver como variáveis influenciam a ordem da lista resultante:
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: `Vamos olhar outro exemplo`

File: pt/lessons/specifics/ets.md [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+iex> :ets.match_object(:user_lookup, {:"$1", :"_", :"$3"})
+[{"doomspork", "Sean", ["Elixir", "Ruby", "Java"]},
+ {"3100", "", ["Elixir", "Ruby", "JavaScript"]}]
+
+{% raw %}iex> :ets.select(:user_lookup, [{{:"$1", :"_", :"$3"}, [], [:"$_"]}]){% endraw %}
+[{"doomspork", "Sean", ["Elixir", "Ruby", "Java"]},
+ {"spork", 30, ["ruby", "elixir"]}]
+```
+
+Apesar do `select/2` um controle mais fino sobre o que e como recuperamos registos, a sinaxe é um bastante hostil e tende a pior. Para lidar com isso, o módulo ETS inclui `fun2ms/1`, para transformar as funções em *match_specs*. Com `fun2ms/1` podemos criar consultas usando uma sintaxe de função mais familiar.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: `recuperamos registros`

Comment (pragmaticivan) [inline pt/lessons/specifics/ets.md]: `bastante hostil e tende a ser pior`

**Comment by pragmaticivan** [comment]
@psantos10 just updated with some changes. Thanks you so much! o/

**Comment by pragmaticivan** [comment]
By the way, I wrote a post in Portuguese about Open Source Contribution: https://medium.com/@pragmaticivan/forking-workflow-contribuindo-em-projetos-open-source-utilizando-git-parte-2-a1849204d02e#.xjtshbxih

**Comment by psantos10** [comment]
@pragmaticivan I am done with this, I think. You can merge.

**Comment by pragmaticivan** [comment]
@psantos10 please squash the commits

**Comment by psantos10** [comment]
@pragmaticivan Done!

**Comment by pragmaticivan** [comment]
Thanks @psantos10 ! o/

**Pull request merged**